feature-libs/organization/schematics/add-organization/index_spec.ts

```diff
@@ -420,4 +427,69 @@ describe('Spartacus Organization schematics: ng-add', () => {
       });
     });
   });
+
```

Comment:
for the sake of 'completion', put an assertion for the unit order feature module after lin 140